src/core/tests/Test_PatchAssessor.py

```diff
@@ -147,6 +147,30 @@ def test_convert_iso8601_duration_to_total_seconds(self):
         self.assertRaises(Exception, lambda: self.runtime.patch_assessor.convert_iso8601_duration_to_total_seconds('6H5M14S'))
         self.assertRaises(Exception, lambda: self.runtime.patch_assessor.convert_iso8601_duration_to_total_seconds(''))
 
+    def test_write_assessment_perf_logs(self):
+        self.runtime.patch_assessor.start_assessment()
+        self.assertTrue(self.runtime.patch_assessor.stopwatch.start_time is not None)
+        self.assertTrue(self.runtime.patch_assessor.stopwatch.end_time is not None)
+        self.assertTrue(self.runtime.patch_assessor.stopwatch.time_taken is not None)
+        self.assertTrue(self.runtime.patch_assessor.stopwatch.task_details is not None)
+        self.assertTrue(self.runtime.patch_assessor.stopwatch.start_time <= self.runtime.patch_assessor.stopwatch.end_time)
+        self.assertTrue(self.runtime.patch_assessor.stopwatch.time_taken >= 0)
+        task_info = "'{}': '{}'".format(str(Constants.PerfLogTrackerParams.TASK), str(Constants.ASSESSMENT))
```

Comment:
nit: use numbered index placeholder
task_info = "'{0}' : '{1}'".format(str(Constants.PerfLogTrackerParams.TASK), str(Constants.ASSESSMENT))

 in other places also

Reply:
Same comment for Test_PatchInstaller.py

Reply:
Done